db/migrate/20160829134746_remove_atomic_container_providers.rb

```diff
@@ -0,0 +1,47 @@
+class RemoveAtomicContainerProviders < ActiveRecord::Migration[5.0]
```

Comment:
@zgalor Why are we renaming from Atomic to Openshift (as opposed to Kubernetes or something else or even at all)?  I would expect we would just delete any existing ones, which probably isn't anything since they were never really exposed in the first place.